topo_processor/stac/item.py

```diff
@@ -30,6 +31,7 @@ def __init__(self, item_id: str):
         self.collection = None
         self.geometry_poly = None
         self.assets = []
+        self.schema = 'https://schemas.stacspec.org/v1.0.0/item-spec/json-schema/item.json'
```

Comment:
Is this defined in pystac anywhere? could we just reference it from there?

Reply:
I've pushed a change this morning

Reply:
Wow that is kinda frustrating how hard it is to get a schema URI from pystac!  maybe we do just want to hard code them :P

Reply:
I know right!